frontend/src/app/shared/header/header.component.ts

```diff
@@ -1,8 +1,39 @@
-import { Component } from '@angular/core';
-import { Router } from '@angular/router';
+import { Component, OnInit } from '@angular/core';
+import { Meta } from '@angular/platform-browser';
```

Comment:
I'm not seeing this import used anywhere. Please remove it and the reference to it on line 13 if it's not being used.